packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/async_jobs/async_jobs.py

```diff
@@ -103,3 +121,117 @@ async def submit(
     )
     assert isinstance(_result, AsyncJobGet)  # nosec
     return _result
+
+
+_DEFAULT_RPC_RETRY_POLICY: dict[str, Any] = {
+    "retry": retry_if_exception_type(RemoteMethodNotRegisteredError),
```

Comment:
I do not get the exception type. Is this being raised while the service is down?

Reply:
maybe `_DEFAULT_RPC_RETRY_POLICY` should better reflect the purpose

Reply:
yes while the service is done. and I don't get what you mean with your secondaty comment